core-bundle/src/Resources/contao/classes/DataContainer.php

```diff
@@ -1229,6 +1229,23 @@ protected function getCacheTags($table, array $ids = array(), $parentTable = '',
 			$tags[] = $ns . $parentTable . '.' . $parentId;
 		}
 
+		// Trigger the oncachetags_callback
+		if (\is_array($GLOBALS['TL_DCA'][$table]['config']['oncachetags_callback']))
```

Comment:
I'm not sure if I like adding a new callback. @contao/developers Is this still how we do it nowadays?

Reply:
I don't like adding an event here. It's still old fashioned DCA stuff. If we have a new driver, then we might go a different direction.

Reply:
I'm generally fine with the callback principle. The markup also works with our new callback service tags.
However, the callback is strange, because it does not pass the datacontainer object, and (almost?) all other callbacks do that. The callback is also missing some arguments (ptable stuff?) and I would expect to return the tags array (or actually return the tags I want to have merged into the array).

Reply:
There is no DC, you don't need one. You also don't need anything else (ptable). I could adjust the return value but it makes no sense to me.

Reply:
I assume you're right, but it's just kinda inconsistent with most other callbacks. I only question if this is clearly backend related? So is it **only** supposed to set/clear tags on backend operations? Then it totally makes sense in the DCA. But what if we need to clear tags if the member data is edited in the front end?

Reply:
I assume you're right, but it's just kinda inconsistent with most other callbacks. I only question if this is clearly backend related? So is it **only** supposed to set/clear tags on backend operations? Then it totally makes sense in the DCA. But what if we need to clear tags if the member data is edited in the front end?

Reply:
I assume you're right, but it's just kinda inconsistent with most other callbacks. I only question if this is clearly backend related? So is it **only** supposed to set/clear tags on backend operations? Then it totally makes sense in the DCA. But what if we need to clear tags if the member data is edited in the front end?

Reply:
I assume you're right, but it's just kinda inconsistent with most other callbacks. I only question if this is clearly backend related? So is it **only** supposed to set/clear tags on backend operations? Then it totally makes sense in the DCA. But what if we need to clear tags if the member data is edited in the front end?

Reply:
I assume you're right, but it's just kinda inconsistent with most other callbacks. I only question if this is clearly backend related? So is it **only** supposed to set/clear tags on backend operations? Then it totally makes sense in the DCA. But what if we need to clear tags if the member data is edited in the front end?